web/components/video/OwncastPlayer/OwncastPlayer.module.scss

```diff
@@ -29,6 +29,11 @@
     max-height: 250px;
   }
 
+  @include screen(mobileLandscape) {
+    height: 250px;
+    max-height: 250px;
```

Comment:
Since we are setting the height to fixed 250px. Do we also need the `max-height` here?